pyetrade/order.py

```diff
@@ -10,86 +10,87 @@
 
 """
 
-import dateutil.parser
 import logging
 import jxmlease
+import xmltodict
+import dateutil.parser
+from requests import sessions

```

Comment:
Is this import only used for the type hint? You might be able to do it off of `OAuth1Session` object.

Reply:
indeed, changed it to `OAuth1Session.request` in the latest commit.